Documentation/README.md

```diff
@@ -8,7 +8,14 @@ Generally dates are stored as a Unix Epoch times in milliseconds.
 Responses are served in JSON format.
 
 ## UUID Parameters
-All uuid parameters are expected to have the UUID without dashes.
+UUID's will work the same with or without dashes, but it is recommended to format it without dashes.
+
+``Recommended Format: f7c77d999f154a66a87dc4a51ef30d19``
+
+``Unrecommended Format: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``
+

```

Comment:
```suggestion
Methods which require a UUID in the API ask for a Java `UUID` object, which can be built using `UUID#fromString`. **Note: ** This method will **only** accept UUIDs in dashed format

``Acceptable: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``
``Wrong: f7c77d999f154a66a87dc4a51ef30d19``
```

Reply:
Dashes aren't really concidered "wrong"... As said in #326. Also I think it should be a bit-less experienced-sounded for people who are new to the API and have no idea what that means.

Reply:
If they want to use the API they should know what `UUID` objects are, you're being too nice

Reply:
Okay.

Reply:
The example files feature the use of `UUID#fromString` to initialize the API. Also, I said **no** dashes is wrong. And yes, it's wrong. I updated #326 because UUIDs with no dashes actually do not work at all

Reply:
So the UUID without dashes will not work?

Reply:
Look at the issue. I provided proof. **Only** a dashed UUID will work

Reply:
okay